### ISSUE TITLE
Activities 1.7.1 and 1.7.3

### ISSUE THREAD

**Issue by mikeshulman** [active-calculus/active-calculus-single-mbx#255]
In these activities, it's not clear to me, or to my students, how one is supposed to deduce the value of f(2) from the graph.  I mean, I suppose the most sensible guess is f(2) = -2.5, since that is the left-hand limit while the right-hand limit doesn't exist, but that still feels like just a guess.  If I were drawing this graph I would have been inclined to include a large filled dot indicating the value of f(2), like the ones for f(-2) and f(-1).

Also, while I'm at it, it doesn't seem quite right to say in the preamble to 1.7.1 that this function is "defined on -4 < x < 4", when in fact it is not defined at 3 (right?).

**Comment by davidfarmer** [comment]
In 1.7.1 it is not possible to determine the value of f(2),
which is okay because the question does not ask for that value.

In part b is asks for f(a) for those "a" where lim_{x\to a} f(x) exists.
That limit does not exist (because the graph wiggles), so the question
does not ask for f(2).

I agree that the hole in the graph at 3 makes it appear the function
is not defined on all of (-4,4).  Or maybe the value of f(3) is not in
the range shown in the graph?  This could become a feature if
the question also asked:  What can you determine about f(3)?
Answer:  it is larger than 4 or smaller than -4.

**Comment by mikeshulman** [comment]
This appears to have been fixed?

**Comment by mattboelkins** [comment]
Yes, I fixed this in some late summer 2022 edits, I think, and forgot to close the issue.  As I'm cleaning some things up now for the 2nd edition to release in August 2025, I made one minor edit but the substantive stuff raised above had been taken care of.  Thanks.
